apps/tangle-cloud/src/pages/instances/Instances/PendingInstanceTable.tsx

```diff
@@ -201,39 +165,61 @@ export const PendingInstanceTable: FC<PendingInstanceTabProps> = ({
             );
           },
         }),
-        columnHelper.accessor('ownerAccount', {
+        columnHelper.accessor('owner', {
           header: () => 'Deployer',
           cell: (props) => {
+            const owner = props.row.original.owner;
+            const ownerUrl = network.createExplorerAccountUrl(owner);
+
             return (
               <TableCellWrapper>
-                {!props.row.original.ownerAccount ? (
+                {!ownerUrl ? (
                   EMPTY_VALUE_PLACEHOLDER
                 ) : (
-                  <Link
-                    to={
-                      network.createExplorerAccountUrl(
-                        isEvmAddress(props.row.original.ownerAccount)
-                          ? props.row.original.ownerAccount
-                          : toSubstrateAddress(props.row.original.ownerAccount),
-                      ) ?? ''
-                    }
-                    target="_blank"
-                    rel="noopener noreferrer"
-                  >
-                    <Button variant="link" className="uppercase body4">
-                      {shortenString(props.row.original.ownerAccount)}
-                    </Button>
-                  </Link>
+                  <>
+                    <Avatar
+                      sourceVariant="address"
+                      value={owner.toString()}
+                      theme="substrate"
+                      size="md"
+                    />
+                    <Link
+                      to={
+                        network.createExplorerAccountUrl(
+                          isEvmAddress(props.row.original.owner)
+                            ? props.row.original.owner
+                            : toSubstrateAddress(props.row.original.owner),
+                        ) ?? ''
+                      }
+                      target="_blank"
+                      rel="noopener noreferrer"
+                    >
+                      <Button variant="link" className="uppercase body4">
+                        {operatorIdentityMap?.get(owner)?.name ??
+                          shortenString(owner)}
+                      </Button>
+                    </Link>
```

Comment:
To render a link directly, you can pass `href` and `target` into the button component.

```tsx
<Button href={accountUrl} target='_blank' variant="link">

</Button>
```